bert_parscit.py

```diff
@@ -15,13 +15,27 @@
 
 BASE_OUTPUT_DIR = "result"
 BASE_TEMP_DIR = "temp"
+BASE_CACHE_DIR = "cached"
+
+device = torch.device("cuda" if torch.cuda.is_available() else "cpu")
```

Comment:
can we use a way similar to training to indicate cpu/gpu usage?
`# train on CPU
python train.py trainer=cpu

# train on 1 GPU
python train.py trainer=gpu`

Reply:
To add an option in each function?

Reply:
Could you check whether pytorch lightning support `python test.py XX=gpu`?

Reply:
By the way, the parent template write config for trainer, so they can use like `python train.py trainer=gpu` instead of `python train.py trainer.accelorator=gpu`. No need for us

Reply:
> Could you check whether pytorch lightning support `python test.py XX=gpu`?

It's ok. I know what you mean. But I think this way can't be applied to `bert_parscit` where we load the model without using `Trainer`. I'm not sure how to set the device when we import `bert_parscit`.